src/android/java/com/pspdfkit/cordova/action/annotation/GetAnnotationsAction.java

```diff
@@ -0,0 +1,109 @@
+package com.pspdfkit.cordova.action.annotation;
+
+import android.annotation.SuppressLint;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.pspdfkit.annotations.Annotation;
+import com.pspdfkit.annotations.AnnotationType;
+import com.pspdfkit.cordova.CordovaPdfActivity;
+import com.pspdfkit.cordova.PSPDFKitCordovaPlugin;
+import com.pspdfkit.cordova.action.BasicAction;
+import com.pspdfkit.document.PdfDocument;
+
+import org.apache.cordova.CallbackContext;
+import org.apache.cordova.PluginResult;
+import org.json.JSONArray;
+import org.json.JSONException;
+
+import java.util.EnumSet;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+import static com.pspdfkit.cordova.Utilities.convertJsonNullToJavaNull;
+
+public class GetAnnotationsAction extends BasicAction {
```

Comment:
One more. I think it would also be beneficial to mention which actions are synchronous vs. asynchronous.